hydra-node/src/Hydra/API/ServerOutput.hs

```diff
@@ -199,7 +199,7 @@ prepareServerOutput ServerOutputConfig{txOutputFormat, utxoInSnapshot} response
         handleTxOutput
           ( key "snapshot"
               . key "confirmedTransactions"
-              .~ toJSON (txToCbor <$> confirmed snapshot)
+              .~ toJSON (confirmed snapshot)
           )
           encodedResponse
```

Comment:
Should: make this now only a `encodedResponse`

This and the other `ConfirmedSnapshot` cases now don't contain transactions and hence do not need to be transformed.